Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectCollectionViewCell.swift

```diff
@@ -10,4 +10,19 @@ import UIKit
 
 class ProjectCollectionViewCell: UICollectionViewCell {
     @IBOutlet weak var projectCellLabel: UILabel!
+    
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        super.touchesBegan(touches, with: event)
+        self.backgroundColor = UIColor(colorLiteralRed: 0xF5/0xFF, green: 0xF5/0xFF, blue: 0xF5/0xFF, alpha: 1)
```

Comment:
Turn red? I wasn't sure if we had anything designed for this is I just freestyles the UI. It is only supposed to turn light grey when pressed. I am not married to it if we find something that looks better.